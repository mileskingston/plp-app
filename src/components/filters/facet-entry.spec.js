import React from 'react';
import renderer from 'react-test-renderer';
import FacetEntry from './facet-entry';

describe('FacetEntry Component', () => {
  const entry = {
    label: 'facet entry label',
    count: 2,
    value: 'facet entry value'
  };

  it('renders component', () => {
    let component = renderer.create(<FacetEntry entry={entry} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});