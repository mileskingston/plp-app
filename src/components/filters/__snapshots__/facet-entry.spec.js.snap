// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FacetEntry Component renders component 1`] = `
<label
  className="facet-entry"
  ufor="facet entry label"
>
  <input
    className="facet-entry__input"
    id="facet entry label"
    name="facet entry label"
    tabIndex="0"
    type="checkbox"
    value="facet entry value"
  />
  <span
    className="facet-entry__label"
  >
    facet entry label
     
    <span>
      2
    </span>
  </span>
</label>
`;
